run_experiments.py

```diff
@@ -32,7 +32,7 @@ def make_dir(path: str) -> None:
 def load_config(config_path: str) -> dict:
     with open(config_path, "r", encoding="utf-8", errors="ignore") \
             as config_file:
-        config_dict = json.loads(config_file.read())
+        config_dict = json.loads(config_file.read(), strict=False)
```

Comment:
Comment why we need the non-strict version?

Reply:
The json file might contain `\r` `\n` `\t` in practice, due to different editors. We should parse them without failure IMO.